### ISSUE TITLE
Question about config for GT reconstruction

### ISSUE THREAD

**Issue by haiphamcse** [weiyithu/SurroundOcc#44]
Hi there, loved your work. I want to ask about the config.yaml for groundtruth generation of nuscenes. When i change the config to the following:
`'depth': 10
'min_density': 0.1
'n_threads': -1
'downsample': False
'voxel_size': 0.4
'max_nn': 20
'pc_range':  [-40, -40, -1, 40, 40, 5.4]
'occ_size':  [200, 200, 16]
'self_range': [3.0, 3.0, 3.0]`
Which changes the pc_range and voxel_size, i saw that the output was no longer correct. Have your team encountered this problem before?
Original GT: "https://drive.google.com/file/d/1x_rXif3HO_-0_lVuIduNQ7QfLgKofw3Z/view?usp=sharing"
Altered GT: "https://drive.google.com/file/d/1sd_FTcMDw_WvE26ei1zsHCavXePiCoIv/view?usp=sharing"

**Comment by weiyithu** [comment]
Hi, do you use the nuscenes data or your own data?

**Comment by haiphamcse** [comment]
Yes the pictures i shown above are from nuscenes. I am currently using the generation code for nuscene as well

**Comment by weiyithu** [comment]
I doubt it maybe due to the pc range. -1 is too high for the z-axis lower bound of nuscenes data since the origin of LiDAR coordinate is on the top of the car but not the ground. If you set -1 as the lower bound, you may loss many points.

**Comment by haiphamcse** [comment]
Thanks for the reply! However i want to ask for the original z bound [-5, 3], did your team choose this empirically or do you base on some prior work?

**Comment by weiyithu** [comment]
It is based on BEVFormer's [config](https://github.com/fundamentalvision/BEVFormer/blob/2c889a36c71fa54534032b955ef0463526155951/projects/configs/bevformer/bevformer_base.py#L11).

**Comment by haiphamcse** [comment]
Thanks for the comment! I want to follow up with 1 last question. Have your team measure the time of the generate_occupancy_nuscenes.py script? How long does it take to generate the entire trainval set of nuscenes and what hardware did your team used?
